GoogleForCreators/web-stories-wp
# Use only one set of color presets for text, shape, page background

## Summary
Uses single set of color presets for all elements.


## Relevant Technical Choices
- Removes `textColors` completely for now since these will likely not be used anymore.


## To-do



## User-facing changes



## Testing Instructions
I
1. Add a Text element.
2. Add some color presets.
3. Add a Shape
4. Verify the same color presets are available for the Shape as well.

II
1. Add a Shape element.
2. Add a color preset with gradient
3. Add a Text element
4. Verify that the same color presets are present for the Text, however, the preset with gradient is grayed out and can't be used.



---



Fixes #2585
